PlateSpace/ETL/platespaceIndexes.js

```diff
@@ -0,0 +1,15 @@
+db.getMongo();
```

Comment:
we might not need this file if we are doing the indexes in `index.js` (see other note above)